src/layout/Body.js

```diff
@@ -93,9 +99,16 @@ Body.defaultProps = {
 };
 
 function mapStateToProps({ technologies }) {
+    let active = technologies && technologies.activeTechnologies;
```

Comment:
This is too complicated, no need to use lodash's filter. If you want to make sure it won't fail on undefined arrays we can update the `mapStateToProps` function to:
```JS
function mapStateToProps({ technologies = { aciveTechnologies = [] } }) {
  return {
    technologies: technologies && technologies.aciveTechnologies.filter(({ disabled }) => !disabled)
  }
}
```